README.md

```diff
@@ -69,6 +70,19 @@ To produce a signed URL, you must enable secure URLs on your source and then pro
 
 ```
 
+## Disabled Path Encoding
+
+Path encoding is enabled by default. It can be toggled off by setting `disable_path_encoding` to `True`:
+
+```python
+>>> from imgix import UrlBuilder
+>>> ub = UrlBuilder("demo.imgix.net", disable_path_encoding=True)
+>>> ub.create_url("file+with%20some+crazy?things.jpg", {'w': 100, 'h': 100})  
+'https://demo.imgix.net/file+with%20some+crazy?things.jpg?h=100&w=100'
```

Comment:
I think this example could be a little clearer. Why don't we include some invalid path characters to demonstrate that this won't get encoded?

```suggestion
>>> ub.create_url(" <>[]{}|^%.jpg", {'w': 100, 'h': 100})  
'http://sdk-test.imgix.net/ <>[]{}|^%.jpg?h=100&w=100'
```